supplementary_style_guide/glossary_terms_conventions/general_conventions/g.adoc

```diff
@@ -280,11 +280,11 @@
 [discrete]
 [[grub]]
 ==== image:images/yes.png[yes] GRUB (noun)
-*Description*: "GRUB" is an acronym for "GRand Unified Bootloader", which is a Linux boot loader.
+*Description*: "GRUB" is an acronym for "GRand Unified Bootloader", which is a Linux boot loader. (Note: GRUB 2 has replaced what was formerly known as GRUB (i.e. version 0.9x), which has, in turn, become GRUB Legacy.)
```

Comment:
Is there a way to avoid the parentheses, and the parentheses within the parentheses? Does that sentence even need to be a Note? If not, maybe remove the `Note:` as well as the parentheses. Or if needs to be Note admonition, then use an Asciidoc Note admonition.